DesignerTest/ViewDesignTests.cs

```diff
@@ -74,5 +78,64 @@ public void ViewDesign_LoadProducts_Count()
             List<Product> products = ViewDesignViewModel.LoadProducts();
             Assert.AreEqual(3, products.Count);
         }
+
+        [Test]
+        [TestCase("0,0|500,0|500,500|0,500", ExpectedResult = 4)]
+        [TestCase("0,0|250,0|250,250|500,250|500,500|0,500", ExpectedResult = 6)]
+        public int ViewDesign_ConvertPosititionsToCoordinates_ShouldReturnList(string positions)
+        {
+            List<Coordinate> coordinates = ViewModel.ConvertPosititionsToCoordinates(positions);
+
+            foreach(Coordinate coordinate in coordinates)
+            {
+                Assert.IsNotNull(coordinate.X);
+                Assert.IsNotNull(coordinate.Y);
+            }
+
+            return coordinates.Count;
+        }
+
+        [Test]
+        [TestCase(10, 10, ExpectedResult = true)]
+        [TestCase(100, 100, ExpectedResult = true)]
+        [TestCase(510, 500, ExpectedResult = false)]
+        [TestCase(150, 270, ExpectedResult = true)]
+        [TestCase(270, 150, ExpectedResult = false)]
+        [TestCase(270, 270, ExpectedResult = true)]
+        public bool ViewDesign_CheckRoomCollisions_ReturnBoolean(int x, int y)
+        {
+            List<Coordinate> coordinates = ViewModel.ConvertPosititionsToCoordinates("0,0|250,0|250,250|500,250|500,500|0,500");
```

Comment:
Wellicht de input als input meegeven

Reply:
Is niet nodig aangezien de laatste parameter wordt gecheckt door NUnit zelf omdat er een bool wordt returned